crypto/ml_kem/ml_kem.c

```diff
@@ -1532,6 +1532,7 @@ int decap(uint8_t secret[ML_KEM_SHARED_SECRET_BYTES],
         secret[i] = constant_time_select_8(mask, Kr[i], failure_key[i]);
     OPENSSL_cleanse(decrypted, ML_KEM_SHARED_SECRET_BYTES);
     OPENSSL_cleanse(Kr, sizeof(Kr));
+    OPENSSL_cleanse(&mask, sizeof(mask));
```

Comment:
This isn't going to do anything useful.

All this is doing is extending the lifetime of the value and forcing it into memory and clearing that -- without this change, I doubt the value would even reach memory.

Reply:
I would also suggest NOT doing this. This byte, would typically be a register-only value, but even if it spills onto the stack, would be difficult to isolate. It is not even directly a secret value, rather this can only be useful if a chosen-ciphertext attacker was trying to recover a non-ephemeral key, and had access to multiple crash dumps that contain this byte. That's a bridge too far I think.